vtds_application_openchami/private/config/config.yaml

```diff
@@ -94,7 +89,7 @@ application:
       # the configuration automatically if it is null. Override it at your
       # own risk if you want a known value.
       redfish_username: root
```

Comment:
REVIEWERS: this is another item I want to handle better. If there is a place I can stuff this in OpenCHAMI itself, that would be cool. Then I could simply generate it and let privileged folks fetch it and use it. For now it is isn't particularly sensitive, since you have to be logged into the management node on the cluster to use it and that requires pretty high level access to GCP instances, but it looks bad. Again, this is a VSHA-700 thing, I think.